src/lib.rs

```diff
@@ -56,14 +60,15 @@ impl Config {
     /// ## Arguments
     ///
     /// - `manifest_path`: The path to the `Cargo.toml` of the crate being documented
-    pub fn new(manifest_path: PathBuf, assets: Vec<Asset>) -> Result<Config> {
+    pub fn new(quiet: bool, manifest_path: PathBuf, assets: Vec<Asset>) -> Result<Config> {
```

Comment:
maybe this should be an enum of levels instead; quiet, verbose, and normal?

Reply:
I agree. What if in the future we want `-vvv`? Having an enum would make that easier and is more type safe. It'll express what it does better.